### PR TITLE
Add fruit icons/improve metadata

### PR DESCRIPTION
**Pull request by danielbayley**
Related to #1149, and #1289.

### CODE REVIEW

**Comment by jguddas** [comment]
Do you have the full apple that you based your `apple-core` icon on?

I don't really like our current `apple` and think it would be cool to have this toggleable, like an `apple-off` icon 😄

**Comment by karsa-mistmere** [comment]
> Do you have the full apple that you based your `apple-core` icon on?
> 
> I don't really like our current `apple` and think it would be cool to have this toggleable, like an `apple-off` icon 😄

Yeah, me neither, I made it and I hate it and already have a new version in my drafts:
image

File: icons/avocado.svg
Comment (jguddas) [inline icons/avocado.svg:1]: Feels quite `pair`ish.

File: icons/fruit.svg
Comment (jguddas) [inline icons/fruit.svg:1]: Super cute ❤️

File: icons/lemon.svg
Comment (jguddas) [inline icons/lemon.svg:1]: Feels a bit week compared to the others and what about adding some dots?

File: icons/peach.svg
Comment (jguddas) [inline icons/peach.svg:1]: Is it possible to make it more circular?

File: icons/strawberry.svg
Comment (jguddas) [inline icons/strawberry.svg:1]: The straight lines need more swoosh.

File: icons/watermelon.svg
Comment (jguddas) [inline icons/watermelon.svg:1]: The rind feels quite big.

File: icons/avocado.svg
Comment (danielbayley) [reply icons/avocado.svg:1]: It’s using the same shape :) Actually, it came first, before `pear`…

File: icons/peach.svg
Comment (danielbayley) [reply icons/peach.svg:1]: I used the top half of `apple` and the bottom half is _literally_ a semi-circle, so not sure how… :)

File: icons/watermelon.svg
Comment (danielbayley) [reply icons/watermelon.svg:1]: It’s modelled exactly on  https://lucide.dev/icons?search=citrus (again, aiming for consistency).

I do have a variation… Although I think it really should have seeds :/

Untitled
<br/>
Artboard 1@2x

**Comment by danielbayley** [comment]
> Do you have the full apple that you based your `apple-core` icon on?

https://lucide.dev/icons?search=apple, of course…

> I don't really like our current `apple`

> Yeah, me neither, I made it and I hate it and already have a new version in my drafts:

Yeah I did think it could do with a facelift… But happy not to have to bother if you already have it @karsa-mistmere ? I’ll need it as a base for `apple-core`…

> it would be cool to have this toggleable, like an `apple-off` icon 😄

@jguddas Like what I was going for with #1163 (which has a much more obvious use case, and I think needs looking at again, now I added some curvature… @ericfennis @karsa-mistmere See https://github.com/lucide-icons/lucide/pull/1163#issuecomment-1529144557)

**Comment by karsa-mistmere** [comment]
apple

File: icons/lemon.svg
Comment (danielbayley) [reply icons/lemon.svg:1]: Added some sheen (waxed)… But to compare:

Untitled
<br/>
Artboard 1@2x

**Comment by danielbayley** [comment]
> apple

@karsa-mistmere Thanks, like the shape! But not sure about the stork… Would it not be better as a curve?

**Comment by jguddas** [comment]
Where I'm from apples are not so top-heavy, looks weird to me but when doing an image search you see a lot of apples that look like this.

Something like this is what I'm used to.

image

**Comment by danielbayley** [comment]
> Where I'm from apples are not so top-heavy, looks weird to me but when doing an image search you see a lot of apples that look like this.
> 
> Something like this is what I'm used to.

Yeah same. But then we don’t eat icons :)

**Comment by karsa-mistmere** [comment]
I think let's use the more geometric stem for apple, peach etc., the curved one makes the top part a bit too dense, especially with a 3px stroke.

As for some others:
- `pineapple-ring`, `pomodoro` and `citrus-2` are not easily recognisable
- at first I thought `citrus-2` was supposed to be a slice of kiwi (maybe it should be?)
- for `strawberry` I preferred the dots as seeds as opposed to lines
- `watermelon` seems to have some optimization issues, and since it's practically identical to the watermelon icon [I originally designed](https://github.com/lucide-icons/lucide/pull/680) I don't feel bad for saying out loud that it just doesn't look very good I'm afraid 😅 
  As long as we can't use any tricks to make the seeds smaller and get away with less than 2px gaps, I don't think we can properly make this work:
  image
- maybe the avocado stone could be smaller?
  image

**Comment by karsa-mistmere** [comment]
Yeah, let's not have the avocado stone off grid, it would look super blurry at 1x.

**Comment by karsa-mistmere** [comment]
How about something like this for strawberry?

strawberry

File: icons/apple.svg [hunk 1]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M12 20.94c1.5 0 2.75 1.06 4 1.06 3 0 6-8 6-12.22A4.91 4.91 0 0 0 17 5c-2.22 0-4 1.44-5 2-1-.56-2.78-2-5-2a4.9 4.9 0 0 0-5 4.78C2 14 5 22 8 22c1.25 0 2.5-1.06 4-1.06Z" />
-  <path d="M10 2c1 .5 2 2 2 5" />
+  <path d="M14 2a2 2 0 0 0-2 2v2" />
+  <path d="M2 11a6 6 0 0 1 10-4.5A6 6 0 0 1 22 11c0 4-4 11-7 11-1.5 0-1.5-1-3-1s-1.5 1-3 1c-3 0-7-7-7-11" />
```
Comment (karsa-mistmere) [inline icons/apple.svg:13]: Over-optimised:
```suggestion
  <path d="M14 2a2 2 0 0 0-2 2v2.53" />
  <path d="M8 5c1.54 0 2.94.58 4 1.53A6 6 0 0 1 22 11c0 4-4 11-7 11-1.5 0-1.5-1-3-1s-1.5 1-3 1c-3 0-7-7-7-11a6 6 0 0 1 6-6Z" />
```

File: icons/apple-core.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M14 2a2 2 0 0 0-2 2v2" />
+  <path d="M15 16a3.9 3.9 0 0 1 1.5-3.1 4 4 0 0 1 4-5.8 5.85 5.85 0 0 0-8.5-.6 5.85 5.85 0 0 0-8.5.6c.2-.1.3-.1.5-.1a4 4 0 0 1 3.5 5.9 3.9 3.9 0 0 1-1.7 7C6.8 21.2 8 22 9 22c1.5 0 1.5-1 3-1s1.5 1 3 1c1 0 2.2-.8 3.2-2.1-1.8-.3-3.2-2-3.2-3.9" />
```
Comment (karsa-mistmere) [inline icons/apple-core.svg:13]: Very over-optimised:
```suggestion
  <path d="M14 2a2 2 0 0 0-2 2v2.53" />
  <path d="M12 6.53a5.98 5.98 0 0 0-8.5.5 4 4 0 0 1 4.02 5.86 4 4 0 0 1-1.76 7.04C6.82 21.17 7.97 22 9 22c1.5 0 1.5-1 3-1s1.5 1 3 1c1.03 0 2.18-.83 3.24-2.07a4 4 0 0 1-1.76-7.03 4 4 0 0 1 4.02-5.87 5.99 5.99 0 0 0-8.5-.5Z" />
```

File: icons/avocado.svg [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 7a4.95 4.95 0 0 0-8.6-3.4c-1.5 1.6-1.6 1.8-5 2.6a8 8 0 1 0 9.4 9.5c.7-3.4 1-3.6 2.6-5 1-1 1.6-2.3 1.6-3.7" />
+  <circle cx="10" cy="14" r="3.5" />
```
Comment (karsa-mistmere) [inline icons/avocado.svg:13]: This circle is still off grid and blurry. :(

File: icons/garlic.svg [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M15 4a2 2 0 0 0-2-2h-2a2 2 0 0 0-2 2v1c0 1-.5 2-1.4 2.5L5.1 9.1A7 7 0 0 0 9 22h6a7 7 0 0 0 3.8-12.8l-2.5-1.6A3.32 3.32 0 0 1 15 5Z" />
+  <path d="M9 5c0 4-2 4-2 9 0 4.4 2.2 8 5 8s5-3.6 5-8c0-5-2-5-2-9" />
```
Comment (karsa-mistmere) [inline icons/garlic.svg:13]: To me this doesn't quite look quintessentially garlicky, the top and the bottom part both look off, here are a few ideas in a similar vein:
image

File: icons/lemon.svg
Comment (karsa-mistmere) [reply icons/lemon.svg:1]: I'd go with either the plain one or the one with two dots to illustrate texture. The one with sheen begs the question why there isn't any sheen on our apple etc.

File: icons/onion.svg [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="2" />
+  <circle cx="12" cy="12" r="6" />
+  <path d="M2.8 8.1a10 10 0 1 0 5.3-5.3C5 4 3 2 3 2L2 3s2 2 .8 5.1" />
+  <path d="M18 20v2" />
+  <path d="m21 21-1.9-1.9" />
+  <path d="M22 18h-2" />
```
Comment (karsa-mistmere) [inline icons/onion.svg:17]: I like how it's all nice and geometrical, but... it also doesn't look like the cross section of any onion that grows on Earth.

These rough drafts of mine are admittedly highly inferior in quality and geometrical perfectness, but I still think an onion should look something more like these.
image

**Pull request merged**